### ISSUE TITLE
Failed to execute script pyi_rth_qt4plugins

### ISSUE THREAD

**Issue by JSantos1985** [pyinstaller/pyinstaller#2207]
Packing applications with PyQt4 on Python 3.5 (64bit), Windows 10 with Pyinstaller 3.2 is resulting in errors when executed in other machines. Here is a minimalist app to illustrate this:

```
from PyQt4 import QtCore, QtGui
app = QtGui.QApplication([])
view = QtGui.QMainWindow()
view.show()
app.exec_()
```

The resulting .exe works fine on this computer but on others throws the following error:

```
Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py", line 41, in <module>
ImportError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py", line 43, in <module>
  File "c:\python35x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 573, in load_module
ImportError: DLL load failed: The specified module could not be found.
Failed to execute script pyi_rth_qt4plugins
```

The error message is unclear since the PySide except exists only to check if it should be packed or not. I added two print statements to pyimod03_importers.py to identify what files were causing the error; it occurs when attempting to load PyQt4.QtCore, which corresponds to the file PyQt4.QtCore.cp35-win_amd64.pyd, which is correctly placed in the package/folder. The warnings file suggests that this might be due to submodule PyQt4.QtCore.QCoreApplication:

```
missing module named PyQt4.QtCore.QCoreApplication - imported by PyQt4.QtCore, c:\python35x64\lib\site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py
missing module named PySide.QCoreApplication - imported by PySide, c:\python35x64\lib\site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py
missing module named PySide - imported by c:\python35x64\lib\site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py
missing module named resource - imported by posix, D:\Dropbox\Projects\SmartQDA\test.py
missing module named posix - imported by os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _datetime.__doc__ - imported by _datetime, datetime, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _posixsubprocess - imported by subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.open - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.BufferedWriter - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.BytesIO - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.DEFAULT_BUFFER_SIZE - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.BufferedRWPair - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.BufferedRandom - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.IncrementalNewlineDecoder - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.TextIOWrapper - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.StringIO - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.BufferedReader - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.FileIO - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.BlockingIOError - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _io.UnsupportedOperation - imported by _io, io, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _heapq._heapreplace_max - imported by _heapq, heapq, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _heapq._heapify_max - imported by _heapq, heapq, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _heapq._heappop_max - imported by _heapq, heapq, D:\Dropbox\Projects\SmartQDA\test.py
missing module named 'org.python' - imported by pickle, D:\Dropbox\Projects\SmartQDA\test.py, xml.sax
missing module named 'org.python'.imp - imported by 'org.python', xml.sax
missing module named 'org.python'.PyStringMap - imported by 'org.python', pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.dumps - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.load - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.Pickler - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.PicklingError - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.PickleError - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.UnpicklingError - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.Unpickler - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.dump - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _pickle.loads - imported by _pickle, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named inspect.CO_GENERATOR - imported by inspect, bdb, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _opcode.stack_effect - imported by _opcode, opcode, D:\Dropbox\Projects\SmartQDA\test.py
missing module named readline - imported by cmd, code, pdb, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.is_builtin - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp._fix_co_filename - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.init_frozen - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.lock_held - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.get_frozen_object - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.is_frozen_package - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.is_frozen - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.acquire_lock - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.release_lock - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _imp.create_dynamic - imported by _imp, imp, D:\Dropbox\Projects\SmartQDA\test.py
excluded module named _frozen_importlib - imported by importlib, importlib.abc, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _frozen_importlib_external - imported by importlib._bootstrap, importlib, importlib.abc, D:\Dropbox\Projects\SmartQDA\test.py
missing module named zipimport.zipimporter - imported by zipimport, pkgutil, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.Union - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.Structure - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.Array - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._Pointer - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.CFuncPtr - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.__version__ - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.RTLD_GLOBAL - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.RTLD_LOCAL - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.ArgumentError - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.FormatError - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.FUNCFLAG_CDECL - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.FUNCFLAG_USE_LASTERROR - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.FUNCFLAG_PYTHONAPI - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.FUNCFLAG_USE_ERRNO - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.LoadLibrary - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.FUNCFLAG_STDCALL - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.dlopen - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.sizeof - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.resize - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.alignment - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.addressof - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.byref - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.get_errno - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.set_errno - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._SimpleCData - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._pointer_type_cache - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.POINTER - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._check_HRESULT - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.get_last_error - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes.set_last_error - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._cast_addr - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._memmove_addr - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._memset_addr - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._string_at_addr - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ctypes._wstring_at_addr - imported by _ctypes, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winreg - imported by platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winreg.OpenKeyEx - imported by _winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winreg.HKEY_LOCAL_MACHINE - imported by _winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winreg.QueryValueEx - imported by _winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winreg.CloseKey - imported by _winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _scproxy._get_proxy_settings - imported by _scproxy, urllib.request
missing module named _scproxy._get_proxies - imported by _scproxy, urllib.request
missing module named _scproxy - imported by urllib.request
missing module named 'java.lang'.Exception - imported by 'java.lang', xml.sax._exceptions
missing module named xml.parsers.expat.ParserCreate - imported by xml.parsers.expat, plistlib, D:\Dropbox\Projects\SmartQDA\test.py
missing module named java - imported by platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named java.System - imported by java, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named 'java.lang' - imported by platform, D:\Dropbox\Projects\SmartQDA\test.py, xml.sax._exceptions
missing module named vms_lib - imported by platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named termios - imported by tty, D:\Dropbox\Projects\SmartQDA\test.py, getpass
missing module named _ssl.OPENSSL_VERSION_NUMBER - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.OPENSSL_VERSION_INFO - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.OPENSSL_VERSION - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.MemoryBIO - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl._SSLContext - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.SSLEOFError - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.SSLWantWriteError - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.SSLWantReadError - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.SSLSyscallError - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.SSLZeroReturnError - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.SSLError - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.CERT_REQUIRED - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.CERT_OPTIONAL - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.CERT_NONE - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.txt2obj - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.nid2obj - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.RAND_pseudo_bytes - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.RAND_add - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.RAND_bytes - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.RAND_status - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.RAND_egd - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.HAS_ECDH - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.HAS_SNI - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.HAS_NPN - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.HAS_ALPN - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl._OPENSSL_API_VERSION - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.enum_crls - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _ssl.enum_certificates - imported by _ssl, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named time.strptime - imported by time, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named winreg.OpenKeyEx - imported by winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named winreg.HKEY_LOCAL_MACHINE - imported by winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named winreg.QueryValueEx - imported by winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named winreg.CloseKey - imported by winreg, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _hashlib.pbkdf2_hmac - imported by _hashlib, hashlib, D:\Dropbox\Projects\SmartQDA\test.py
missing module named hashlib.sha512 - imported by hashlib, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _lzma._encode_filter_properties - imported by _lzma, lzma, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _lzma._decode_filter_properties - imported by _lzma, lzma, D:\Dropbox\Projects\SmartQDA\test.py
missing module named grp - imported by tarfile, shutil, D:\Dropbox\Projects\SmartQDA\test.py
missing module named grp.getgrnam - imported by grp, shutil, D:\Dropbox\Projects\SmartQDA\test.py
missing module named socket.AF_INET - imported by socket, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named socket.SOCK_STREAM - imported by socket, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named socket.SOL_SOCKET - imported by socket, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named socket.SO_TYPE - imported by socket, ssl, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _operator.__doc__ - imported by _operator, operator, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _bz2.BZ2Compressor - imported by _bz2, bz2, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _bz2.BZ2Decompressor - imported by _bz2, bz2, D:\Dropbox\Projects\SmartQDA\test.py
missing module named unicodedata.ucd_3_2_0 - imported by unicodedata, stringprep, encodings.idna, D:\Dropbox\Projects\SmartQDA\test.py
missing module named binascii.a2b_qp - imported by binascii, quopri, D:\Dropbox\Projects\SmartQDA\test.py
missing module named binascii.b2a_qp - imported by binascii, quopri, D:\Dropbox\Projects\SmartQDA\test.py
missing module named binascii.b2a_base64 - imported by binascii, email.base64mime, D:\Dropbox\Projects\SmartQDA\test.py
missing module named binascii.a2b_base64 - imported by binascii, email.base64mime, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _sre.MAXREPEAT - imported by _sre, sre_constants, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _sre.MAXGROUPS - imported by _sre, sre_constants, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.getweakrefs - imported by _weakref, weakref, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.proxy - imported by _weakref, weakref, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.ProxyType - imported by _weakref, weakref, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.CallableProxyType - imported by _weakref, weakref, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.getweakrefcount - imported by _weakref, weakref, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.ReferenceType - imported by _weakref, weakref, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _weakref.ref - imported by _weakref, _weakrefset, weakref, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _functools.reduce - imported by _functools, functools, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _functools.cmp_to_key - imported by _functools, functools, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _functools.partial - imported by _functools, functools, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _functools._lru_cache_wrapper - imported by _functools, functools, D:\Dropbox\Projects\SmartQDA\test.py
missing module named sys.getwindowsversion - imported by sys, platform, D:\Dropbox\Projects\SmartQDA\test.py
missing module named sys.maxsize - imported by sys, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named posix._exit - imported by posix, os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named pwd.getpwnam - imported by pwd, shutil, D:\Dropbox\Projects\SmartQDA\test.py
missing module named pwd - imported by posixpath, tarfile, shutil, http.server, webbrowser, D:\Dropbox\Projects\SmartQDA\test.py, netrc, getpass
missing module named posix._have_functions - imported by posix, os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named nt._getvolumepathname - imported by nt, ntpath, D:\Dropbox\Projects\SmartQDA\test.py
missing module named nt._getfullpathname - imported by nt, ntpath, D:\Dropbox\Projects\SmartQDA\test.py
missing module named nt._getfinalpathname - imported by nt, ntpath, D:\Dropbox\Projects\SmartQDA\test.py
missing module named nt._isdir - imported by nt, ntpath, D:\Dropbox\Projects\SmartQDA\test.py
missing module named nt._exit - imported by nt, os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named nt._have_functions - imported by nt, os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named ce - imported by os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named ce._exit - imported by ce, os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named ce._have_functions - imported by ce, os, D:\Dropbox\Projects\SmartQDA\test.py
missing module named ntpath.commonprefix - imported by ntpath, unittest.util, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _warnings.filters - imported by _warnings, warnings, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _warnings.warn - imported by _warnings, warnings, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _warnings._filters_mutated - imported by _warnings, warnings, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _warnings.warn_explicit - imported by _warnings, warnings, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _warnings._defaultaction - imported by _warnings, warnings, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _warnings._onceregistry - imported by _warnings, warnings, D:\Dropbox\Projects\SmartQDA\test.py
missing module named time.monotonic - imported by time, subprocess, threading, socketserver, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _thread.allocate_lock - imported by _thread, _strptime, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _thread.get_ident - imported by _thread, reprlib, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _thread.RLock - imported by _thread, functools, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _collections.defaultdict - imported by _collections, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _collections.OrderedDict - imported by _collections, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _collections._count_elements - imported by _collections, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _collections.deque - imported by _collections, threading, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _thread.stack_size - imported by _thread, threading, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _thread._local - imported by _thread, threading, D:\Dropbox\Projects\SmartQDA\test.py
missing module named threading.current_thread - imported by threading, _threading_local, D:\Dropbox\Projects\SmartQDA\test.py
missing module named threading.RLock - imported by threading, _threading_local, bz2, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.ceil - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.log - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.exp - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.pi - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.e - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.acos - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.cos - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.sqrt - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named math.sin - imported by math, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named sys.modules - imported by sys, dummy_threading, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _dummy_threading - imported by dummy_threading, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _dummy_threading.__all__ - imported by _dummy_threading, dummy_threading, D:\Dropbox\Projects\SmartQDA\test.py
missing module named dummy_threading.RLock - imported by dummy_threading, bz2, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.CREATE_NEW_CONSOLE - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.STD_ERROR_HANDLE - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.CREATE_NEW_PROCESS_GROUP - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.STARTF_USESHOWWINDOW - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.STD_INPUT_HANDLE - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.STD_OUTPUT_HANDLE - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.STARTF_USESTDHANDLES - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _winapi.SW_HIDE - imported by _winapi, subprocess, D:\Dropbox\Projects\SmartQDA\test.py
missing module named os.urandom - imported by os, random, D:\Dropbox\Projects\SmartQDA\test.py
missing module named builtins.abs - imported by builtins, operator, D:\Dropbox\Projects\SmartQDA\test.py
missing module named builtins.str - imported by builtins, locale, D:\Dropbox\Projects\SmartQDA\test.py
missing module named builtins.open - imported by builtins, tokenize, tarfile, bz2, D:\Dropbox\Projects\SmartQDA\test.py
missing module named itertools.islice - imported by itertools, threading, reprlib, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named itertools.count - imported by itertools, threading, D:\Dropbox\Projects\SmartQDA\test.py
missing module named itertools.starmap - imported by itertools, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named itertools.chain - imported by itertools, tokenize, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named itertools.repeat - imported by itertools, tokenize, collections, D:\Dropbox\Projects\SmartQDA\test.py
missing module named copy.deepcopy - imported by copy, weakref, email.generator, D:\Dropbox\Projects\SmartQDA\test.py
missing module named org - imported by copy, D:\Dropbox\Projects\SmartQDA\test.py
missing module named org.PyStringMap - imported by org, copy, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _struct._clearcache - imported by _struct, struct, D:\Dropbox\Projects\SmartQDA\test.py
missing module named _struct.__doc__ - imported by _struct, struct, D:\Dropbox\Projects\SmartQDA\test.py
missing module named struct.calcsize - imported by struct, ctypes, D:\Dropbox\Projects\SmartQDA\test.py
missing module named struct.unpack - imported by struct, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named struct.pack - imported by struct, pickle, D:\Dropbox\Projects\SmartQDA\test.py
missing module named errno.ENOENT - imported by errno, gettext, D:\Dropbox\Projects\SmartQDA\test.py
```

However, tweaking with the specs to explicitly bundle QtCore.QCoreApplication ends up with having it missing regardless.

The exact same script I provided, when packed with Pyinstaller 3.0 on Python 2.7, works correctly on my test Windows 10 machine with the default .spec settings. I have replicated this issue on a computer with a clean Python 3.5 install with the required libraries installed through pip to make sure this isn't just a problem on my installation's end.

Any ideas on what might be causing this?

**Comment by dzimbeck** [comment]
having a similar issue with images not displaying on other computers. In my warnings I'm unable to import pyside. Installing it with pip doesn't fix the issue. My software works on many computers but some it won't show images. It may have something to do with qt4 plugins since pixmap has a set of plugins associated with that file

**Comment by vdfebook** [comment]
Same thing then JSantos with PyQt4, win10, python 3.5.2 (32bits) and Pyinstaller developpement.
On other computer (win7) just have a message box : Failed to execute script pyi_rth_qt4plugins.
The app cannot start.

**Comment by dbinette** [comment]
compiled on windows10 python 3.5.2, compiles and runs file (pyside is not used or installed)

but when running on client PC:
c:\Program Files (x86)\mbioc>mbioc.exe
Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py", line 41
, in <module>
ImportError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth_qt4plugins.py", line 43
, in <module>
  File "c:\python35\lib\site-packages\PyInstaller\loader\pyimod03_importers.py",
 line 573, in load_module
ImportError: DLL load failed: The specified module could not be found.
Failed to execute script pyi_rth_qt4plugins

**Comment by Alexander3** [comment]
TLDR; On 64bit system install 64bit python and upgrade pip. (It worked on win7, python 3.4)

I had the same problem, exe generated on win10, python 3.5.1, didn't work on win7.
I've even installed `PySide` and forced PyInstaller to include it with `--hidden-import PySide`, the error message now was:
```
Traceback (most recent call last):
  File "site-packages\PySide\_utils.py", line 93, in get_pyside_dir
  File "c:\lib\python35-32\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 573, in load_module
ImportError: DLL load failed: Nie można odnaleźć określonego modułu.
```

When I tried to run my app from sources on this win7 computer with python 3.5.1 (previously installing PyQt from [Unofficial Windows Binaries for Python](http://www.lfd.uci.edu/~gohlke/pythonlibs/#pyqt4)
the error message was similar:
```
Traceback (most recent call last):
  File "Stegano_gui.py", line 10, in <module>
    from PyQt4 import QtCore, QtGui
ImportError: DLL load failed: Nie można odnaleźć określonego modułu.
```

I guessed that it was because my Python was 32bit on 64bit system, so it installed packages with 32bit DLLs.
Finally I've installed python 3.4 64bit on win7 computer and program started working. I ran pyinstaller from this environment and .exe worked on both systems.

**Comment by Marzybear** [comment]
Same problem for me here, Python 3.5.0 64bit on Windows7 64bit machine for development, run on a windows 7 64bit machine without Python and program crashes with same error as original post.  Using the Pyinstaller current development version 3.3.  Have tried to build exe with and without upx, have also tried including PyQt4 path in the spec file.....nothing works....please help

**Comment by Alexander3** [comment]
@Marzybear maybe it's connected to Python version, try using 3.4

**Comment by dbinette** [comment]
I reverted to python 3.4.4  and pyqt 4, 64 bit
I also stripped out a lot of unnecessary  modules.
finally i was able to use pyinstaller with glitching.

We can now develop on windows 10 and deploy to same or windows 8,  other
platforms not tested.

**Comment by Marzybear** [comment]
Thanks so much!!!! It works on Python 3.4, PyInstaller 3.2

**Comment by cglwn** [comment]
Works with Python 3.5 with PyInstaller 3.2 too. I had trouble installing PyInstaller on Python3.4 because pywin no longer releases binaries for Python 3 versions below 3.5 as per https://github.com/mhammond/pywin32/issues/1172.

I would still would like to see Python 3.6 supported though.

**Comment by BoboTiG** [comment]
Closing as PyQt4 support has been removed since #5118.
